Resources/Prototypes/SS220/Entities/Structures/barricade.yml

```
@@ -0,0 +1,423 @@
+  description: A barricade made out of wood planks. It looks like it can take a few solid hits.
+  parent: BaseStructure
+  name: barricade
+  abstract: true
+  components:
+  - type: InteractionOutline
+  - type: Sprite
+    drawdepth: BlastDoors
+    noRot: false
+  - type: Occluder
+  - type: Appearance
+  - type: Physics
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb {}
+        mask:
+        - FullTileMask
+        layer:
+        - WallLayer
+  
+- type: entity
+  id: EstBaseBarricadeDirectional
+  description: A barricade made out of wood planks. It looks like it can take a few solid hits.
+  parent: EstBaseBarricade
+  name: barricadedirectional
+  abstract: true
+  components: 
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.5,-0.5,0.5,-0.36"
+        mask:
+        - FullTileMask
+        layer:
+        - WallLayer
+        
+- type: entity
+  id: EstBaseBarrucadeFloor
+  description: A barricade made out of wood planks. It looks like it can take a few solid hits.
+  parent: BaseStructure
+  name: barricade
+  abstract: true
+  components: 
+  - type: InteractionOutline
+  - type: Sprite
+    drawdepth: BlastDoors
+    noRot: false
+  - type: Appearance
+  - type: AtmosExposed
+  - type: Physics
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.5,-0.5,0.39,-0.36"
+        mask:
+        - FullTileMask
+        layer:
+        - WallLayer
+ #Баррикады       
+- type: entity
+  id: BarricadeWooden
+  description: Баррикада из уплотнённых деревянных досок. Защищает вас от чужих глаз. 
+  parent: EstBaseBarricade
+  name: деревянная уплотненная баррикада
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Sprite
+    sprite: SS220/Structures/barricades.rsi
+    state: barricade_wooden
+    noRot: false 
+  - type: Construction
+    graph: BarricadeWooden
+    node: barricadefull
+  - type: Tag
+    tags:
+    - Wooden
+  - type: Flammable
+    fireSpread: true
+    damage:
+      types:
+        Heat: 2 
+  - type: FireVisuals
+    sprite: Effects/fire.rsi
+    normalState: 1
+  - type: Damageable
+    damageModifierSet: Wood
+    damageContainer: StructuralInorganic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 90
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          MaterialWoodPlank1:
+            min: 3
+            max: 3
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+        
+- type: entity
+  id: BarricadeSteel
+  description: Баррикада из стальных листов. Защищает вас от чужих глаз. Способен выдержать пулю.
+  parent: EstBaseBarricade
+  name: стальная баррикада
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Sprite
+    sprite: SS220/Structures/barricades.rsi
+    state: barricade_steel
+    noRot: false 
```

```suggestion
    noRot: false 
```
они же по умолчанию и так крутятся, нет? получается можно просто стереть